server/routes/threads.route.js

```diff
@@ -13,10 +13,13 @@ router.route('/')
 router.route('/')
     .get(threadsCtrl.index);
 
-router.route('/:threadId/reply')
+router.route('/thread/:threadId/reply')
```

Comment:
@nickmccally I believe naming the route this way will actually mount it as `/threads/thread/:threadId/reply`. Is there a reason why we would want to name it this way?